src/main/java/com/compilercharisma/chameleonbusinessstudio/dto/RepeatingAppointment.java

```diff
@@ -0,0 +1,47 @@
+package com.compilercharisma.chameleonbusinessstudio.dto;
+
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.util.*;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import lombok.*;
+
+/**
+ * holds an appointment that is meant to repeat on specified days
+ */
+@AllArgsConstructor
+@Builder
+@Data
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RepeatingAppointment {
+    /**
+     * required by Vendia
+     */
+    private String _id;
+
+    /**
+     * if set to false, the appointment contained herein will not be copied when
+     * creating new appointments
+     */
+    private boolean isEnabled;
```

Comment:
Use the `Boolean` wrapper in case we have nulls so the app does not complain about that

Reply:
Good catch! I just assumed it would default to null, but I'll do this.